task1.js

```diff
@@ -0,0 +1,38 @@
+
+var x=6, y=15, z=4;
+  
+function foo(x, y, z) {
+x += y - x++ * z ; 
+ 	console.log('x=' + x);
+ 	z = -- x - y * 5 ; 
+ 	console.log('z=' + z);
+ 	y /= x + 5 % z ;
+ 	console.log('y=' + y);  
+ 	z = x++ + y * 5 ; 
+ 	console.log('z=' + z);
+ 	x = y - x++ * z ; 
+ 	console.log('x=' + x);
+}
+foo (x, y, z);
```

Comment:
1) Make sure that you are using correct code styling, I mean that you use wrong paddings.
2) Calculate these formulas on the paper or somewhere else and compare the results.